app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -163,15 +146,15 @@ def history_database_name
 
   # Adding disks is supported only by API version 4.0
   def with_disk_attachments_service(vm)
-    with_version4_vm_service(vm) do |service|
+    with_vm_service(vm) do |service|
       disk_service = service.disk_attachments_service
       yield disk_service
     end
   end
 
-  def with_version4_vm_service(vm)
-    connection = connect(:version => 4)
-    service = connection.system_service.vms_service.vm_service(vm.uid_ems)
+  def with_vm_service(vm)
+    # connection = connect
```

Comment:
I do not understand this comment, can you explain please?

Reply:
It's my fault, I commented out with the aim to remove before the commit but I forgot it. I'll fix in a while